src/Okta.Sdk.UnitTests/OktaClientShould.cs

```diff
@@ -354,5 +355,35 @@ public void RegisterPayloadHandlers()
             PayloadHandler.ForContentType("application/x-pem-file").GetType().Should().Be(typeof(PemFilePayloadHandler));
             PayloadHandler.ForContentType("application/x-x509-ca-cert").GetType().Should().Be(typeof(X509CaCertPayloadHandler));
         }
+
+        [Fact]
+        public async Task EnrollEmailFactor()
```

Comment:
Can you please create a separate class called `UserFactorsClientShould` and move factors tests there? That way is easy to find a test for a particular client. We did the same for [`ApplicationsClient`](https://github.com/okta/okta-sdk-dotnet/blob/master/src/Okta.Sdk.UnitTests/ApplicationsClientShould.cs) for example.

Reply:
Done